app/routes/vet/dairy-cattle-on-farm.js

```diff
@@ -0,0 +1,45 @@
+const Joi = require('joi')
+const { vetVisitData: { dairyCattleOnFarm } } = require('../../session/keys')
+const getYesNoRadios = require('../helpers/yes-no-radios')
+const session = require('../../session')
+
+const legendText = 'Were there more than 10 dairy cattle on the farm at the time of the review?'
+const radioId = 'dairyCattleOnFarm'
```

Comment:
Same as above, can this be removed and the usages replaced with the `dairyCattleOnFarm` session key please.